src/Containers/GithubCallback.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+
+import { config } from '../config'
+import { getAuthenticatedUser } from '../Utils/auth'
+import { Loader } from '../Components/Common/Loader'
+
+const { enableOAuth, OAuthClientID, OAuthSecret } = config
+
+const GithubCallback = () => {
+  const [loading, setLoading] = useState(true)
+
+  const authenticate = async (code) => {
+    // Apparently we need to use a proxy to make this request (CORS blocked for client-side only applications)
+    const response = await fetch('https://cors-anywhere.herokuapp.com/https://github.com/login/oauth/access_token', {
```

Comment:
Is this some kind of hack.

Reply:
This url allows us to use URL without taking care of CORS (because we can't do that in client-side JS, we need a node server for instance)